.changeset/grumpy-maps-hope.md

```diff
@@ -0,0 +1,10 @@
+---
+'@clerk/clerk-js': minor
+'@clerk/nextjs': minor
+'@clerk/clerk-react': minor
+'@clerk/types': minor
+---
+
+Introduce experimental support for Google One Tap
+- React Component `<__experimental_GoogleOneTap/>`
+- JS `clerk.__experimental_mountGoogleOneTap(node,props)`
```

Comment:
❓ Is the `__experimental_` prefix the one we settled for?

Reply:
I thought it was `experimental_`

Reply:
Pretty sure we settled for `__experimental_` same one we had for the most recent project "passkeys"